shell/sc_gnrc_6ctx strtok_r missing

building `examples/gnrc_border_router` throws out the following error when building:
```
...
"make" -C ./RIOT/sys/shell/commands
./RIOT/sys/shell/commands/sc_gnrc_6ctx.c: In function '_gnrc_6ctx_add':
./RIOT/sys/shell/commands/sc_gnrc_6ctx.c:86:16: error: implicit declaration of function 'strtok_r' [-Werror=implicit-function-declaration]
     addr_str = strtok_r(prefix_str, "/", &save_ptr);
                ^
...
```
tested on Ubuntu 16.04.1 LTS with:
`arm-none-eabi-gcc (GNU Tools for ARM Embedded Processors) 5.3.1 20160307 (release) [ARM/embedded-5-branch revision 234589]`
`andarm-none-eabi-gcc (GNU Tools for ARM Embedded Processors) 5.4.1 20160919 (release) [ARM/embedded-5-branch revision 240496]`

Building the example with a ~3 weeks old RIOT works flawlessly, so probably something has changed in the newlib?

--- a/sys/shell/commands/sc_gnrc_6ctx.c
+++ b/sys/shell/commands/sc_gnrc_6ctx.c
@@ -12,6 +12,8 @@
  * @file
  */
 
+/* Required for strtok_r in string.h, when building with -std=c99 */
+#define _DEFAULT_SOURCE 1
 #include <inttypes.h>
 #include <stdio.h>
 #include <stdlib.h>

